active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,120 @@
+
+# Background
+
+There’s currently no consistent and Fluent way to address the common UX pattern of a breadcrumb.
+This control is needed for situations where the user's navigation trail (in a file system or menu system) needs
+to be persistently visible.
+A Breadcrumb control is movitated by its use in many app scenarios and supporting developers in migrating from WPF.
+
+This control is similar to an ItemsControl (has an ItemsSource property), but does not derive from it.
+
+## V1/V2 Breadcrumb
+
+The work for Breadcrumb has been scoped to V1 and V2 stages of the API.
+In V1 Breadcrumb no chevrons are interactable to show children.
+V2 Breadcrumb will add functionality to have flyouts from chevrons for every node,
+to view children of that node. This spec focuses on V1 Breadcrumb,
+but keeping in mind that V1 is positioned in a way to have V2 features smoothly added later. 
+
+# API Pages
+
+## Breadcrumb class
+
+Use the Breadcrumb control to private persisting 'trail' that shows the user's navigation path.
+
+This is not the 'history' of the user's navigation, but the direct path of pages or folders from
+the root node to their current position. For situations where resizing means the full path can't be shown,
+the Breadcrumb will 'crumble' starting at the root node. In addition,
```

Comment:
Could you define 'crumble' here?
> For situations where resizing means the full path can't be shown, the Breadcrumb control will replace the item text with an ellipsis. This will prioritize the content after the first ellipsis...?
I think my confusion comes with not understanding when exactly the control will crumble or not crumble item text.

Reply:
The breadcrumb items will crumble when there is not enough space to render all of them, hence, 'crumbling' the highest-level (closest to the root) ones to provide more space for the other items to be rendered

Reply:
updating the wording here to clarify this